molecule/README.md

```diff
@@ -22,13 +22,16 @@ test11 | 1            | N                       | As for #5 but then deletes a n
 
 # Local Installation & Running
 
-Local installation on a Centos7 machine looks like:
+Local installation on a CentOS 8 machine looks like:
 
     sudo yum install -y gcc python3-pip python3-devel openssl-devel python3-libselinux
     sudo yum install -y yum-utils
     sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
     sudo yum install -y docker-ce docker-ce-cli containerd.io
-    pip3 install -r molecule/requirements.txt --user
+    python3 -m venv venv
+    . venv/bin/activate
+    pip install -U pip
```

Comment:
python3 -m pip

Reply:
`pip` gets you the pip from the venv here.